convert_fendl.py

```diff
@@ -257,9 +256,8 @@ def check_special_case(particle_details, script_step):
             # Extract files, the fendl release was compressed using type 9 zip format
             # unfortunatly which is incompatible with the standard python zipfile library
             # therefore the following system command is used
-            subprocess.call(['unzip', '-o', f, '-d', extraction_dir])
-    os.chdir(cwd)
-    
+            subprocess.call(['unzip', '-o', download_path / particle / Path(f), '-d', extraction_dir])
```

Comment:
As long as the left-side operand is a `Path`, it's ok for the right-side to be a string
```suggestion
            subprocess.call(['unzip', '-o', download_path / particle / f, '-d', extraction_dir])
```